cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -265,8 +265,13 @@ function(px4_nuttx_add_export)
 		COMMAND ${MKDIR} -p ${nuttx_src}
 		COMMAND ${CP} -a ${CMAKE_SOURCE_DIR}/NuttX/. ${nuttx_src}/
 		COMMAND ${RM} -rf ${nuttx_src}/.git
+						  ${nuttx_src}/apps/.git
```

Comment:
Yes that would be nice. I tried to switch this part to rsync at one point, but there were windows problems. Confusion over cmake windows paths and rsync (cygwin) paths I think. Interested in taking a stab at it using relative paths? For my own sanity I try to avoid doing anything PX4 related on windows these days.